slack-plugin/src/main/kotlin/slack/gradle/ApkVersioningPlugin.kt

```diff
@@ -129,7 +122,7 @@ internal class ApkVersioningPlugin : Plugin<Project> {
           val mappedVersionCodeProvider =
             versionCodeProvider.map { rawCode ->
               @Suppress("MagicNumber")
-              ApkVersioning.VERSION_CODES.getValue(variantOutput.abiString) * 10000000 + rawCode
+              ApkVersioning.VERSION_CODE * 10000000 + rawCode
```

Comment:
Let's drop this `10000000` and the `VERSION_CODE` magic constant and handle it internally in the slack repo's gradle.properties, no reason to impose this heh. Then we can just pass `versionCodeProvider` directly to the `variantOutput.versionCode`

Reply:
Removed the calculation here https://github.com/slackhq/slack-gradle-plugin/pull/503/commits/37159ee0888a58413fae0c74dae5eada22871532

Per this change, we'll need to update the version code in [release.version](https://github.com/tinyspeck/slack-android-ng/blob/main/ci/release.version) to 90012557 right?

Reply:
Yup exactly